### ISSUE TITLE
Display and Backgrounds controls not working properly with nvidia-driver for UHD DP1.2 + MST

### ISSUE THREAD

**Issue by Redsandro** [linuxmint/cinnamon-control-center#142]
Some monitors like the **Dell Ultrasharp UP2414Q** transmit 60 FPS UHD over DP by using [MST](https://en.wikipedia.org/wiki/DisplayPort#Multi-Stream_Transport) to stream two halfs of the screen. 

In Windows and macOS, these 'monitors' are detected and treated as single monitors. In Linux, the monitor

1. is seen as two separate monitors.
1. has left and right side are swapped.
1. needs nvidia option `TwinView` to be seen as one single monitor
1. does not work with the `cinnamon-settings display` app in this mode, throwing the following error:  
    > GDBus.Error:org.freedesktop.DBus.Error.UnknownMethod: No such interface 'org.cinnamon.SettingsDaemon.XRANDR_2' on object at path /org/cinnamon/SettingsDaemon/XRANDR   

5. Concequently, cannot properly show those beautiful wallpapers from the _"Backgrounds"_ app, because these are automatically cut according to the (swapped) cinnamon monitor configuration.

Theoretical solution, either:

* Cut wallpapers according to the order of monitors in `nvidia-settings` if that driver is in use, or
* Automatically copy order of monitors from `nvidia-settings` to `cinnamon-settings display`, or
* Let user change order of monitors in `cinnamon-settings display` just so wallpapers get rendered properly

---

Here's a screenshot I hope illustrates the problem. There are two monitors attached, a **Dell Ultrasharp UP2414Q** showing up as two half-sized monitors, and a smaller **Samsung SyncMaster 172v** also showing a broken wallpaper after being scaled down to compensate for the 2x sized UHD GUI. Click for full size.

![image](https://cloud.githubusercontent.com/assets/1702193/16049176/098fc012-3257-11e6-85d5-a7117804d210.png)

As you can see, the `nvidia-settings` shows the proper order is `DFP-2.9` left, `DFP-2.8` 
right. However `cinnamon-settings display` insists it is the other way around. Swapping them - which is hard to do because of bugs similar to #135 - cannot be applied because it results in the aforementioned error message, visible in the dialog to the right.

**Comment by leigh123linux** [comment]
Did you try enabling DP1.2 in the monitor settings?

**Comment by Redsandro** [comment]
@leigh123linux yes, I think I didn't explain correctly. DP1.2 is working fine, otherwise I wouldn't be able to have 60FPS on UHD, which I do have now.

The problem emerges when using DP1.2 + MST.

**Comment by leigh123linux** [comment]
@Redsandro I am not able to debug this issue here as my 4k monitor is SST

**Comment by annosresearch** [comment]
you have no monitor.xml in your .conf directory.
You can recreate it with arandr - simpler than with xrandr. ;)

**Comment by icarter09** [comment]
@Redsandro are you still experiencing this issue?

**Comment by Redsandro** [comment]
@icarter09 I've replaced my monitor because of this issue. IIRC I still had this issue in Mint 19.1, but haven't tested 19.3 of 20 beta.

Not sure if this issue should be closed because it "no longer affects me", or remain open until someone with MST can confirm presence or absence of the issue.
